webapp/lib/view.dart

```diff
@@ -1137,6 +1147,10 @@ class ConversationSummary with LazyListViewItem {
     _checkboxHidden = !show;
     if (_selectCheckbox != null) _selectCheckbox.hidden = !show;
   }
+  void _showWarning(bool show) {
+    _warning = true;
```

Comment:
`_warning = show;` ?

Reply:
Yes, sorry, fixed now, thanks for catching this, it would have been quite a pain to debug!